int32 hue
int32 hue_tol
int32 min_value
int32 min_saturation
bool dontcare
